engine/c.cpp

```diff
@@ -383,9 +382,76 @@ KVDKStatus KVDKListErase(KVDKEngine* engine, KVDKListIterator* pos) {
   return engine->rep->ListErase(pos->rep);
 }
 
-KVDKStatus KVDKListPut(KVDKEngine* engine, KVDKListIterator* pos,
-                       char const* elem_data, size_t elem_len) {
-  return engine->rep->ListPut(pos->rep, StringView{elem_data, elem_len});
+KVDKStatus KVDKListReplace(KVDKEngine* engine, KVDKListIterator* pos,
+                           char const* elem_data, size_t elem_len) {
+  return engine->rep->ListReplace(pos->rep, StringView{elem_data, elem_len});
+}
+
+KVDKStatus KVDKListMultiPushFront(KVDKEngine* engine, char const* key_data,
+                                  size_t key_len, char const* const* elems_data,
```

Comment:
`char const * const *`?

Reply:
char const* means string that should not be modified.
char const** means an array of strings.
You should not modify the underlying pointers, so we decorate char const* as char const* const*.